migrations/migration.template

```diff
@@ -1,9 +1,9 @@
 import * as knex from 'knex'
 
-exports.up = async (knex: Knex): Promise<void> => {
+export async function up (knex: Knex): Promise<void | Knex.SchemaBuilder> {
 
 }
 
-exports.down = function(knex: Knex): Knex.SchemaBuilder {
+export async function down (knex: Knex): Promise< void | Knex.SchemaBuilder> {
```

Comment:
I was wrong the return value should be Knex.SchemaBuilder, not Promise: https://github.com/knex/knex/blob/master/types/index.d.ts#L1528